twilight-interactions-derive/src/command/model/create_command.rs

```diff
@@ -42,8 +42,11 @@ pub fn impl_create_command(input: DeriveInput, fields: Option<FieldsNamed>) -> R
     };
     let name_localizations = localization_field(&attributes.name_localizations);
     let description = match &attributes.desc {
-        Some(desc) => desc.clone(),
-        None => parse_doc(&input.attrs, span)?,
+        Some(desc) => quote! { ::std::option::Option::Some((#desc).to_string()) },
+        None => match parse_doc(&input.attrs, span).ok() {
```

Comment:
We should change the `parse_doc` signature to  `Result<Option<String>, _>` instead, errors should not be ignored silently.